cypress/integration/home.spec.ts

```diff
@@ -1,34 +1,16 @@
 /// <reference path="../support/index.d.ts" />
 
-// import fake from 'faker'
+describe('Test desktop element visibility', () => {
+  beforeEach(() => {
+    cy.clearLocalStorage()
```

Comment:
I'm guessing this (`cy.clearLocalStorage()`) was taken from the login test but can probably also be removed until we implement some more comprehensive tests